src/app/selectors.js

```diff
@@ -141,7 +141,7 @@ const getDecodedMacaroons = (macaroons) => {
 const getModelUUIDByName = (name, modelData) => {
   let owner = null;
   let modelName = null;
-  if (name.includes("/")) {
+  if (name?.includes("/")) {
```

Comment:
Why this was being called without a model name?

Reply:
The test was calling it without a model name before I set up the params provider.